src/raftstore/store/store.rs

```diff
@@ -1869,7 +1871,9 @@ impl<T: Transport, C: PdClient> mio::Handler for Store<T, C> {
                     error!("{} handle raft message err: {:?}", self.tag, e);
                 }
             }
-            Msg::RaftCmd { request, callback } => self.propose_raft_command(request, callback),
+            Msg::RaftCmd { send_time, request, callback } => {
+                self.propose_raft_command(send_time, request, callback)
```

Comment:
seem that you can record the wait time here directly and no need to pass to propose_raft_command